lib/Logger.js

```diff
@@ -49,6 +54,40 @@ class Logger {
         Config.update({ dump: levelName });
     }
 
+    getLogLevel() {
+        return Config.level;
+    }
+
+    getDumpLevel() {
+        return Config.dump;
+    }
+
+    /**
+     *  This function creates a new Logger, from another one,
+     *  changing its submodule field.
+     *
+     *  @param {Logger} baseLogger - Base Logger to create a new one.
+     *  @param {String} submodule - The name of the submodule.
+     *  @return {Logger} - The new logger with the new submodule.
+     */
+    static updateSubmodule(baseLogger, submodule) {
```

Comment:
I planned to pass the Logger from submodule to submodule, but your comments make me realize it's the same things as creating a new one in each submodules ...
So you're right, it's quite useless.